tasks/sep/ink-001-fp-holocene-pectra-upgrade/NestedSignFromJson.s.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {NestedSignFromJson as OriginalNestedSignFromJson} from "script/NestedSignFromJson.s.sol";
+import {Simulation} from "@base-contracts/script/universal/Simulation.sol";
+import {Types} from "@eth-optimism-bedrock/scripts/libraries/Types.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {LibString} from "solady/utils/LibString.sol";
+import {GnosisSafe} from "safe-contracts/GnosisSafe.sol";
+import {Vm, VmSafe} from "forge-std/Vm.sol";
+import "@eth-optimism-bedrock/src/dispute/lib/Types.sol";
+import {DisputeGameFactory} from "@eth-optimism-bedrock/src/dispute/DisputeGameFactory.sol";
+import {FaultDisputeGame} from "@eth-optimism-bedrock/src/dispute/FaultDisputeGame.sol";
+import {PermissionedDisputeGame} from "@eth-optimism-bedrock/src/dispute/PermissionedDisputeGame.sol";
+import {MIPS} from "@eth-optimism-bedrock/src/cannon/MIPS.sol";
+import {ISemver} from "@eth-optimism-bedrock/interfaces/universal/ISemver.sol";
+
+contract NestedSignFromJson is OriginalNestedSignFromJson {
+    using LibString for string;
+
+    DisputeGameFactory constant dgfProxy = DisputeGameFactory(0x860e626c700AF381133D9f4aF31412A2d1DB3D5d);
+    address constant livenessGuard = 0xc26977310bC89DAee5823C2e2a73195E85382cC7;
+    bytes32 constant absolutePrestate = 0x03dfa3b3ac66e8fae9f338824237ebacff616df928cf7dada0e14be2531bc1f4;
```

Comment:
Where is this prestate from?

Reply:
This is the release candidate that seb cut and the one going into the op-challenger release tomorrow https://github.com/ethereum-optimism/optimism/issues/13627#issuecomment-2627347212